doc/yaml_struct.md

```diff
@@ -0,0 +1,223 @@
+## Currently, 4 types are defined, namely
+1.Gateway
+2.Traffic distribution rules
+3.Define the target service
+4.Plug-in extension
+
+### Gateway
+For edge traffic, extract the host separately and define a Gateway type
+
+```yaml
+kind:Gateway
+name: baidu-gw
+servers:
+ - port:
+     number: 80
+     name: http
+     protocol: HTTP
+   hosts:
+   - "a.domain.com"
+   - "b.domain.com"
+```
+|  object/field   | describition |
+|  ----  | ----  |
+| Gateway  | the type for Edge traffic  |
+| Gateway.servers  | Define edge traffic service list  |
+| Gateway.servers.port  | the port for service  |
+| Gateway.servers.port.protocol  | Specify protocol  |
+| Gateway.servers.hosts  | List of domain names that a certain service can accept  |
+
+### Define traffic distribution rules
+
+```yaml
+kind: Rule
+name: xxx-rules
+hosts:
+- "domain.com"
+gateways:
+- baidu-gw
```

Comment:
choose another name, `baidu` is not good

Reply:
change `baidu` to `foo`